addons/website_event/controllers/main.py

```diff
@@ -90,36 +84,25 @@ def dom_without(without):
         # count by domains without self search
         for date in dates:
             if date[0] != 'old':
-                date[3] = event_obj.search(
-                    request.cr, request.uid, dom_without('date') + date[2],
-                    count=True, context=request.context)
+                date[3] = Event.search_count(dom_without('date') + date[2])
 
         domain = dom_without('type')
-        types = event_obj.read_group(
-            request.cr, request.uid, domain, ["id", "event_type_id"], groupby="event_type_id",
-            orderby="event_type_id", context=request.context)
-        type_count = event_obj.search(request.cr, request.uid, domain,
-                                      count=True, context=request.context)
+        types = Event.read_group(domain, ["id", "event_type_id"], groupby=["event_type_id"], orderby="event_type_id")
+        type_count = Event.search_count(domain)
```

Comment:
don't you think, sum([int(type['event_type_id_count']) for type in types]) == search_count result